app/views/pre_registrations/new.html.erb

```diff
@@ -1,170 +1,162 @@
-            <p>
-                <%= f.check_box :packReady, :checked => false %>
-            </p>
-        </div>
-    </div>
-    <br/>
-    <br/>
-
-
-    <div class="row">
+<br/>
+<p>
+  <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#addEmployee" aria-expanded="false" aria-controls="collapseExample">
+    Add New Employee
+  </button>
+</p>
+<div class="collapse" id="addEmployee">
+    <div class="card card-body">
+        <h3><b>Employee information fill by HR </b></h3><br>
+            <%= form_for @pre_registration do |f| %>
+            <div class="row">
             <div class="col-md-6">
+                <%= f.label :name, 'Employee Name' do %>
+                    <%= 'Employee Name' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
                 <p>
-                    <%= label :companyEmail, 'Company Email' %>
-                    <%= label '', ' *', class: 'required-asterisk' %>
+                    <%= f.text_field :name, :class => 'form-control', required: 'true', :maxlength=>"128"%>
                 </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :joiningDate, 'Joining Date' do %>
+                    <%= 'Joining Date' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
                 <p>
-                    <%= f.text_field :companyEmail ,:class => 'form-control', :maxlength=>"128"%>
+                    <%= f.text_field :joiningDate, :class => 'form-control timesheet-date-picker_new_join', value: Time.now.strftime("%Y/%m/%d"), :onkeydown=>"return false" %>
+                </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :designation, 'Designation' do %>
+                    <%= 'Designation' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.text_field :designation, :class => 'form-control', required: 'true', :maxlength=>"128"%>
+                </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :user_id, 'Name of TTF/Mentor' do %>
+                    <%= 'Name of TTF/Mentor' %>
+                    <span class='required-asterisk'>*</span>
+                <% end %>
+                <p>
+                    <%= f.select :user_id, options_for_select( @ttf_list.collect { | ttf_list | [ttf_list.name,  ttf_list.id] }), {}, {required: 'true', autofocus: 'true', class: 'form-control', id:'select-other-projects'} %>
+                </p>
+            </div>
+  
+            <div class="col-md-6">
+                <%= f.label :emailGroup, 'Suggested Email Group' do %>
+                    <%= 'Suggested Email Group' %>
+                <% end %>
+                <p>
+                    <%= f.text_field :emailGroup ,:class => 'form-control', required: 'true',:maxlength=>"128"%>
```

Comment:
Change hash syntax. Put space after comma

Reply:
changed apu
please check it once again just to make sure whether I've used proper syntax or not